docs/extend/modular-resources/_index.md

```diff
@@ -64,9 +64,9 @@ To get started working with modular resources:
 
 - Browse the [Viam registry](https://app.viam.com/registry) to see modules uploaded by other users.
 
-- [Deploy a module](/extend/modular-resources/configure/) to your robot from the Registry.
+- [Deploy a module](/extend/modular-resources/configure/) to your smart machine from the Registry.
 
-- Browse the [modular resources tutorials](/extend/modular-resources/examples/) for examples of deploying and using custom modular resources on your robot.
+- Browse the [modular resources tutorials](/extend/modular-resources/examples/) for examples of deploying and using custom modular resources on your smart machines.
```

Comment:
For all of these ones that talk about deploying to a robot/smart machine, I wonder if robot is still the right word to use since that’s an organizational piece in the app still

Reply:
Could be - I was thinking on the high level page we say smart machines and then we switch over to robot once the UI is involved